skype/skype-ps/skype/Get-CsTeamsAudioConferencingPolicy.md

```diff
@@ -0,0 +1,107 @@
+---
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/en-us/powershell/module/skype/get-csteamsaudioconferencingpolicy
+schema: 2.0.0
+---
+
+# Get-CsTeamsAudioConferencingPolicy
+
+## SYNOPSIS
+
+Audio conferencing policies can be used to manage audio conferencing toll- and toll-free numbers to be displayed in meeting invites created by users within your organization.
+
+## SYNTAX
+
+### Identity (Default)
+```
+Get-CsTeamsAudioConferencingPolicy [[-Identity] <String>] [-LocalStore] [<CommonParameters>]
+```
+
+### Filter
+```
+Get-CsTeamsAudioConferencingPolicy [-LocalStore] [-Filter <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The CsTeamsAudioConferencingPolicy cmdlet enables administrators to control audio conferencing toll- and toll-free numbers to be displayed in meeting invites created by users within your organization. The Get-CsTeamsAudioConferencingPolicy cmdlet enables you to return information about all the audio-conferencing policies that have been configured for use in your organization.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+PS C:\> Get- CsTeamsAudioConferencingPolicy
+```
+
+The command shown in Example 1, Get-CsTeamsAudioConferencingPolicy is called without any additional parameters; this returns a collection of all the teams audio conferencing policies configured for use in your organization.
+
+### EXAMPLE 2
+```
+PS C:\> Get-CsTeamsAudioConferencingPolicy -Identity "EMEA Users"
+```
+
+The command shown in Example 2, Get-CsTeamsAudioConferencingPolicy is used to return the per-user audio conferencing policy that has an Identity “EMEA Users”. Because identities are unique, this command will never return more than one item.
+
+## PARAMETERS
+
+### -Filter
+
+```yaml
+Type: System.String
+Parameter Sets: Filter
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Identity
+Unique identifier for the policy to be retrieved. To retrieve the global policy, use this syntax: -Identity global. To retrieve a per-user policy use syntax similar to this: -Identity “EMEA Users”.
+If this parameter is not included, the Get-CsTeamsAudioConferencingPolicy cmdlet will return a collection of all the teams audio conferencing policies configured for use in your organization.
+
+```yaml
+Type: System.String
+Parameter Sets: Identity
+Aliases:
+
+Required: False
+Position: 2
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -LocalStore
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
```

Comment:
```suggestion
Type: SwitchParameter
```
We use the short version of the Type value in cmdlet documentation please review other occurrences